shared/src/business/useCaseHelper/initialFilingDocuments/updateInitialFilingDocuments.test.js

```diff
@@ -156,6 +164,7 @@ describe('addNewInitialFilingToCase', () => {
     const mockNewRQT = {
       ...mockRQT,
       docketEntryId: applicationContext.getUniqueId(),
+      isOnDocketRecord: true,
```

Comment:
It looks like this attribute is being automatically set now in the `addInitialFilingCase` method that's called from `updateInitialFilingDocuments`, so could we remove this setting here?

Reply:
Update incoming!

Reply:
Updated!